package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "avatars-utils",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
Can you revert this change too, please? We'll do this post-merge as part of publishing; doing it before then is confusing for posterity and not entirely truthful.

Reply:
Yes! I blew away the commit that bumped the version.